src/include/mpitimpl.h

```diff
@@ -1316,6 +1316,10 @@ typedef struct MPIR_T_event_instance_s {
     MPI_Count timestamp;
     void *data;
 } MPIR_T_event_instance_t;
+#ifdef HAVE_MPIT_EVENTS
 void MPIR_T_event_instance(int event_index, MPI_T_cb_safety cb_safety, void *data);
+#else
+#define MPIR_T_event_instance(...)
+#endif
```

Comment:
This is a bit mysterious. How does it work?

Reply:
So right now you can either enable or disable events globally. When disabled (default), the event instance function calls become empty macros, as they are in performance critical paths.

Reply:
I see. I was not familiar with the `...` syntax.